404 Not Found on https://github.com/adnanrahic/node-gcstats/archive/refs/tags/1.5.8.tar.gz

We've suddenly started having this error when trying to install modules, I can see this release is in npm but not in github releases. Any idea what might have caused this?

```
npm ERR! node-pre-gyp http GET https://github.com/adnanrahic/node-gcstats/archive/refs/tags/1.5.8.tar.gz
npm ERR! node-pre-gyp ERR! install response status 404 Not Found on https://github.com/adnanrahic/node-gcstats/archive/refs/tags/1.5.8.tar.gz 
npm ERR! node-pre-gyp WARN Pre-built binaries not installable for @sematext/gc-stats@1.5.8 and node@16.19.1 (node-v93 ABI, unknown) (falling back to source compile with node-gyp) 
npm ERR! node-pre-gyp WARN Hit error response status 404 Not Found on https://github.com/adnanrahic/node-gcstats/archive/refs/tags/1.5.8.tar.gz
```

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "1.5.9",
   "description": "Exposes node v8 garbage collection stats",
   "main": "index.js",
-  "repository": "git@github.com:adnanrahic/node-gcstats.git",
+  "repository": "git@github.com:sematext/node-gcstats.git",
   "license": "Unlicense",
   "engines": {
     "node": ">=6"
@@ -37,7 +37,7 @@
     "module_path": "./build/{module_name}/v{version}/{configuration}/{node_abi}-{platform}-{arch}-{libc}",
     "remote_path": "./",
     "package_name": "{version}.tar.gz",
-    "host": "https://github.com/adnanrahic/node-gcstats/archive/refs/tags/"
+    "host": "https://github.com/sematext/node-gcstats/archive/refs/tags/"
   },
   "release-it": {
     "github": {

